---
title: "Notable Projects"
---

## **Characterization of the healthy Indian gut microbiome**

This study delves into the distinct features of the gut microbiota in healthy Indian individuals, a subject that has been surprisingly understudied despite India having the world's largest population. Utilizing 16S rRNA gene sequencing and comprehensive bioinformatics analysis, the research highlights significant compositional and phylogenetic variations in the Indian gut microbiome. The investigation correlates these variations with the biogeographic separation of individuals in urban and rural settings.

The study identifies specific microbial genera, including *Prevotella*, *Lactobacillus*, *Bifidobacterium*, and *Megasphaera*, as notably abundant in the Indian gut microbiome. These findings are further linked to distinctive dietary patterns prevalent in India, particularly the consumption of fermented foods. A meta-analysis compares the Indian gut microbiome with populations from Spain, Korea, Bangladesh, and the United States, revealing that Indians share the highest microbial taxa with their neighboring Bangladeshi population. This decreasing similarity with American, Spanish, and Korean populations suggests a reflection of shared ethnicity and lifestyle factors.

The most intriguing result emerges from the meta-analysis, indicating only three common microbial taxa across all populations studied. This suggests that the gut microbiome of geographically separated populations is uniquely individualistic, with a minimal number of microbial taxa contributing to the core global microbiome. Additionally, the study notes the similarity between the Indian gut microbiome and that of mammals with an omnivorous diet. This observation is noteworthy given the mixed vegetarian and non-vegetarian dietary habits prevalent in India, emphasizing the significant impact of diet on shaping gut microbial communities.

In summary, the research underscores the distinctive nature of the Indian gut microbiome compared to other major global populations. The findings lay the foundation for larger cohort studies on the Indian Gut Microbiome, aiming to explore specific bacterial taxa or microbial functions as potential targets for medical intervention studies.

The major results from this project were published in Frontiers in Microbiology, for more details please read [here](https://www.frontiersin.org/articles/10.3389/fmicb.2016.00660/full#h5).

## **Characterization of the gut microbiome of Indian diabetic subjects**

This research involved two distinct cohorts of individuals, encompassing both healthy and diabetic subjects. In the first cohort, the study focused on perturbations in the gut microbiota of newly diagnosed and known diabetic individuals in India. Considering the unique genetic, nutritional, developmental, and socio-economic aspects of diabetes in India, the assessment of gut microbiota becomes crucial in understanding its potential role in influencing these characteristics.

The study identified specific bacterial taxa, such as *P. copri*, *Lachnospiraceae*, and *Ruminococcaceae*, as significantly abundant in healthy subjects, while diabetic individuals exhibited increased abundance of *Firmicutes* and *Lactobacillus*. Bacterial interaction network analysis revealed disease-state-specific patterns, indicating individual bacterial responses to diabetes. The abundance of dominant archaea, *Methanobrevibacter* and *Methanosphaera*, exhibited opposite trends in healthy and diabetic subjects. Fungi, particularly opportunistic genera like *Aspergillus*, *Candida*, and *Saccharomyces*, were enriched in newly diagnosed diabetic subjects. The decline in butyrate-producing bacteria was observed in correlation with diabetes risk factors. Variations in gut microbiota were associated with factors like fasting glucose, high triglycerides, low HDL, and fasting insulin, with specific interactions linked to the stage of glucose intolerance. Using PICRUSt, the study predicted metabolic disturbances in the gut microbiome of newly diagnosed diabetic subjects, highlighting deficiencies in essential functions.

In the second diabetes cohort, nine differentially abundant genera were reported, with *Akkermansia*, *Blautia*, and *Ruminococcus* decreased and *Lactobacillus* increased in newly diagnosed diabetic subjects. Machine learning analysis identified discriminative factors, including *Akkermansia*, *Sutterella*, and serum metabolites like fasting glucose, HbA1c, methionine, and total antioxidants. Interestingly, the microbial diversity of newly diagnosed diabetics, but not prediabetic subjects, significantly differed from healthy individuals. After antidiabetic treatment, the microbial diversity of known diabetic patients appeared to recover and resemble that of healthy subjects. Overall, these findings shed light on the dynamic relationship between gut microbiota and diabetes across different stages and contribute valuable insights for further investigation and potential interventions.

Findings from this project were published in [Frontiers in Microbiology](https://www.frontiersin.org/articles/10.3389/fmicb.2017.00214/full) and in [ASM mSystems](https://journals.asm.org/doi/full/10.1128/msystems.00578-19).

## **The gut microbiome in urolithiasis**

Using 16S rRNA gene sequencing, this project assesses the gut microbiota in individuals with recurrent kidney stones and compares it with healthy individuals. Targeted metagenomic sequencing of the formyl-CoA transferase (frc) gene and qPCR assays are employed to monitor active OMBS and their association with hyperoxaluria.

The study reveals that hyperoxaluric subjects have reduced urine volume and elevated oxalate levels in their urine compared to healthy subjects. Bioinformatics analysis of 16S rRNA gene sequence data indicates significant microbial changes in patients with recurrent kidney stones. The overall diversity of OMBS in the human gut increases significantly in kidney stone subjects under the selective pressure of hyperoxaluria, leading to the specific enrichment of acid-tolerant pathobionts with oxalate-utilizing abilities. Surprisingly, there is a decreased abundance of the well-studied oxalate-metabolizing bacterium *Oxalobacter formigenes* in kidney stone patients.

Using PICRUSt-based metagenomic imputation, the study reveals the enrichment of enzymes participating in oxalate metabolism in kidney stone patients. Additionally, two oxalic acid and bile acid-tolerant isolates of *Lactobacillus plantarum* are isolated from healthy human stool samples, suggesting their potential as probiotics for managing metabolic disorders such as hyperoxaluria.

In summary, this research sheds light on the intricate relationship between gut microbiota, oxalate metabolism, and hyperoxaluria, providing valuable insights into potential probiotic interventions for managing specific metabolic disorders associated with oxalate accumulation.

The findings are published in [Scientific Reports](https://www.nature.com/articles/srep34712%3C).

## ***Clostridioides difficile*** **infection in small rodent models**

*Clostridioides difficile*, an opportunistic diarrheal pathogen, is a major cause of antibiotic-associated and hospital-acquired diarrhea, constituting over 70% of healthcare-related gastrointestinal infections. CDI symptoms vary from mild diarrhea to severe conditions like ulcerative colitis and toxic megacolon, resulting from disruptions in the resident gut microbiota. Despite the significant impact of diet on human health and its relationship with the microbiota, research on the effects of diet on CDI has been limited. Western diets and popular weight-loss diets, known to induce substantial changes in the gut microbiome, have conflicting literature regarding their impact on CDI.

Using a mouse model with the hypervirulent strain of *C. difficile* R20291 (RT027) and antibiotic-induced CDI, this study examined disease outcomes and microbial community dynamics in mice fed high-fat diets compared to a high-carbohydrate diet and a standard rodent diet. The two high-fat diets, particularly an Atkins-like high-fat/high-protein diet, exacerbated CDI, resulting in severe cases and 100% mortality. A high-fat/low-protein, medium-chain-triglyceride (MCT)-like diet led to highly variable CDI outcomes. Surprisingly, mice on a high-carbohydrate diet were protected from CDI, despite the diet's high refined carbohydrates and low fiber content. Bacteria from the families *Lachnospiraceae* and *Ruminococcaceae* decreased in abundance due to diet and/or antibiotic treatment, members of these bacterial families compete with *C. difficile* for amino acids and protected healthy animals from CDI in the absence of antibiotics.

The study suggests that antibiotic treatment may lead to the loss of *C. difficile* competitors, creating a favorable environment for its proliferation and virulence. Moreover, the detrimental effects of high-fat/high-protein diets intensify these outcomes, while high-carbohydrate diets may offer protective effects irrespective of carbohydrate source or antibiotic-driven loss of *C. difficile* competitors.

You can read more about this study in article published in [ASM mSystems](https://journals.asm.org/doi/full/10.1128/msystems.00765-19) and in [ASM Microbiology Spectrum](https://journals.asm.org/doi/full/10.1128/spectrum.01804-21).

## **The gut microbiome of pupfish of Mojave Desert**

In the Mojave Desert of the Southwestern United States, isolated geothermally heated springs house unique populations of pupfishes in the genus *Cyprinodon*. Notably, Devils Hole in Death Valley National Park serves as a habitat where these pupfishes have evolved physiological adaptations to thrive in the high temperature and salinity of the water. Specifically, these pupfishes exhibit paradoxical anaerobism, entering this state in response to both endogenously produced and exogenously supplied ethanol in a dose-dependent manner.

This study explores the role of gut microbiota in ethanol-associated paradoxical anaerobism by depleting microbial communities with antibiotics and conducting 16S rRNA gene sequencing. The analysis revealed a dominance of bacterial phyla such as Proteobacteria, Fusobacteria, Bacteroidetes, Firmicutes, Actinobacteria, Patescibacteria, and Dependentiae in the pupfish gut, consistent with other fish gut microbiota. Interestingly, compared to the control group, fish with depleted microbiota spent more time in paradoxical anaerobism. Bioinformatics analysis highlighted the removal of dominant gut flora, including the genus *Cetobacterium*, as a factor leading to increased duration of paradoxical anaerobism. The ethanol-utilizing abilities of Cetobacterium, particularly through alcohol dehydrogenase action, were implicated. The study proposes that *Cetobacterium* may act as an ethanol sink, and depleting it using antibiotics might result in increased acetaldehyde accumulation in fish tissues, leading to the closure of VDAC and eventually triggering paradoxical anaerobism.

The results are published in [Animal Microbiome](https://animalmicrobiome.biomedcentral.com/articles/10.1186/s42523-020-00037-5).

## **Assessment of gut microbiome of frugivorous bat**

Bats play a crucial role in ecosystem functioning, offering various essential services to the environment. While bats are widely recognized as carriers of pathogenic viruses, their potential role as carriers of pathogenic bacteria remains underexplored. This study focused on Rousettus leschenaultii, a frugivorous bat species roosting in the Robber's Cave, Mahabaleshwar, Maharashtra, India.

A culture-based approach, utilizing four bacteriological media, was employed to isolate and identify over a thousand bacteria from the gut of Rousettus leschenaultii. Identification was achieved through 16S rRNA gene sequencing and BLAST analysis at NCBI, revealing a prevalence of bacteria from the family Enterobacteriaceae, including putative pathogens. The pathogenic potential of the most frequently cultivated bacterium, Escherichia coli, was assessed, uncovering the presence of virulent factors in many isolated E. coli strains.

Next-generation sequencing characterized the bacterial communities in bat guano samples, highlighting significant inter-individual variation in microbial diversity. Interestingly, higher bacterial diversity was observed in decaying guano compared to freshly defecated samples. The sequencing data identified 31 different bacterial phyla in bat guano, emphasizing the extensive uncultivated bacterial communities in this environment. Alignment of 16S data against a human pathogenic bacteria database revealed a small proportion of sequences associated with well-known human pathogens, including Yersinia pestis, Brucella melitensis, and Mycobacterium tuberculosis. The study suggests that this bat species may carry potential bacterial pathogens, emphasizing the need to investigate the effects of these pathogens on bats and their potential transmission to humans and other animals.

The results are published in [Scientific Reports](https://www.nature.com/articles/srep36948).
